### ISSUE TITLE
How can I convert shap values to probability increase/decreases?

### ISSUE THREAD

**Issue by NFeruch** [shap/shap#2783]
Really cool library, still trying to wrap my brain around it.

So currently, I know how to convert the base (expected) value from log odds to probability, with
```
odds = np.exp(explainer.expected_value)
odds / (1 + odds)
```
This works fine, but the problem comes when I try and convert each individual shap value to a probability increase/decrease. That formula doesn't work, so I'm wondering how I can get the percent increase/decrease that each feature contributes

![image](https://user-images.githubusercontent.com/35023657/205425262-0d0aa082-07eb-4481-bf03-72dfb0b5d9bb.png)

Basically, what percent do each of the lengths (like the length I annotated in red on the picture) take up?

**Comment by thatlittleboy** [comment]
Making shap values explain in terms of probability isn't really possible (some level of approximation is required) if the raw output of the model is not itself probabilities.
E.g.
> For sklearn random forests shap values (+expected value) do sum to the probability because that is what the trees
output. But for XGBoost they sum to the margin output of the model, ...

Read the thread here: https://github.com/slundberg/shap/issues/29#issuecomment-371877433 and https://github.com/slundberg/shap/issues/168 for more context.

If you so happen to be using TreeExplainer with "interventional", there's a model_output="probability" you can try.
